.travis.yml

```diff
@@ -58,6 +58,7 @@ env:
     - "TEST_SUITE=spec:legacy DB=postgres"
 
 before_install:
+  - "gem install bundler"
```

Comment:
probably you should still include comment like that there...

So that future oliver knows:

* why this line was neccessary in the dark past
* and can decide if it is still neccessary in the bright future